### ISSUE TITLE
Source json files

### ISSUE THREAD

**Issue by marwaneltoukhy** [The-OpenROAD-Project/OpenLane#1538]
### Description

In the new `config.json` we need to have a way to source another `config.json`

**Comment by mattvenn** [comment]
yes, I depend on this for Zero to ASIC group submissions and TinyTapeout

**Comment by vijayank88** [comment]
In [MPW CI](https://github.com/The-OpenROAD-Project/OpenLane-MPW-CI) in top level we've `config.tcl` and source caravel flow `config.tcl`. https://github.com/The-OpenROAD-Project/OpenLane-MPW-CI/blob/dfdfdb7052f0b24a0fffaebe08d12ee3517dbf69/scripts/AC_top/config.tcl#L4
In latest OpenLane/ MPW-8 shuttle forced to use `config.json`.
How to source `json` file through `tcl` file?

**Comment by vijayank88** [comment]
@kareefardi 
Any workaround to source `json` file inside `tcl` file?

**Comment by mattvenn** [comment]
@donn can we get an update on this please?

**Comment by donn** [comment]
Not for a while- I'll elaborate in DM.

**Comment by donn** [comment]
Somewhat addressed in https://github.com/efabless/openlane2/pull/351- you can now simply pass multiple JSON files to OpenLane and the config will be the composition of all of these.
